packages/core/src/manifest/Manifest.ts

```diff
@@ -0,0 +1,10 @@
+export interface Manifest {
+  version: string
+  deploy: Deploy[]
+}
+
+export interface Deploy {
+  name: string
+  include: string
+  headers?: object
```

Comment:
jak chcesz typ dla dowolnego objektu to lepiej użyć Record<string, unknown> (object to może być wszystko prawie, nawet null)